### PR TITLE
Pozwalaj na używanie tostera tylko gdy stoi się przy tosterze

### PR DESCRIPTION
**Pull request by MrQubo**

### CODE REVIEW

File: toster.py [hunk 1]
```diff
@@ -39,6 +45,12 @@ def __init__(self, backup_file):
         # on other types of exceptions, application will just crash
 
 
+    def _verify_user_near_toster(self, user):
+        near_toster_users = self.near_toster_channel.members
+        is_near_toster = any(user.id == us.id for us in near_toster_users)
+        if not is_near_toster:
+            raise TosterOopsie('Musisz podejść do tostera!!')
```
Comment (krzys-h) [inline toster.py:52]: Make it more clear to the users - say something like "you need to enter near toster voice channel"

File: toster.py [hunk 2]
```diff
@@ -39,6 +45,12 @@ def __init__(self, backup_file):
         # on other types of exceptions, application will just crash
 
 
+    def _verify_user_near_toster(self, user):
+        near_toster_users = self.near_toster_channel.members
+        is_near_toster = any(user.id == us.id for us in near_toster_users)
```
Comment (krzys-h) [inline toster.py]: I think you can do just `user in self.near_toster_channel.members`? This is how I always checked for mentions

**Review by krzys-h — CHANGES_REQUESTED** [review]

File: toster.py (hunk 2 above)
Comment (enedil) [reply toster.py]: If not, just monkey-patch the class:
```
type(next(near_toster_users)).__eq__ = lambda x, y: x.id == y.id
```

File: toster.py (hunk 1 above)
Comment (MrQubo) [reply toster.py:52]: It can't be too easy.

**Pull request merged**